App Listing Quality

Doomsdayrs opened the pull request:
- Branding
  - [x] [Colors](https://docs.flathub.org/docs/for-app-authors/metainfo-guidelines/quality-guidelines#has-primary-brand-colors): What are "Thrive"s brand colors?
- App Icon: Having an SVG would help edit the logo. Currently the logo are just static PNGs.
![image](https://github.com/flathub/com.revolutionarygamesstudio.ThriveLauncher/assets/38189170/9615f1e4-9094-494e-98f0-0673ea39f26c)
  - [ ] [Fill](https://docs.flathub.org/docs/for-app-authors/appdata-guidelines/quality-guidelines/#reasonable-footprint)
  - [ ] [Contrast](https://docs.flathub.org/docs/for-app-authors/appdata-guidelines/quality-guidelines/#good-contrast)
  - [ ] [Styles](https://docs.flathub.org/docs/for-app-authors/appdata-guidelines/quality-guidelines/#in-line-with-contemporary-styles)
- Summary
  - [x] [Length](https://docs.flathub.org/docs/for-app-authors/appdata-guidelines/quality-guidelines/#not-too-long-1)

Doomsdayrs committed:
Shorten app summary

hhyyrylainen commented:
AFAIK, we have no picked brand colours. We have colours guide for the *game* itself: https://wiki.revolutionarygamesstudio.com/wiki/Visual_Style_Guide but choosing brand colours doesn't seem to be something that anyone has ever thought of for Thrive.

Luckily there is an svg version of the launcher icon:
![launcher_and_almost_thrive_icon](https://github.com/flathub/com.revolutionarygamesstudio.ThriveLauncher/assets/3796411/26e28024-2021-42c6-8d26-b2b59fa3d88f)

Note that the file contains multiple icons so it renders a bit weird unless you use something like inkscape to extract the  one specific icon from it.

Doomsdayrs committed:
Add icon work

Doomsdayrs commented:
> AFAIK, we have no picked brand colours. We have colours guide for the _game_ itself: https://wiki.revolutionarygamesstudio.com/wiki/Visual_Style_Guide but choosing brand colours doesn't seem to be something that anyone has ever thought of for Thrive.

Perhaps something to put as a TODO for whoever is in charge of marketing? For now I can use one of the in game colors?

Doomsdayrs commented:
> Note that the file contains multiple icons so it renders a bit weird unless you use something like inkscape to extract the one specific icon from it.

I made two new files on this repository. I just noticed that the logo is not the same in the PNGs and the SVG. The SVG logo is more of a strict "T" while the PNGs have a stylized "T". Did someone forget to commit work?

hhyyrylainen commented:
Probably best to pick one of the blues / cyan hues in the game style guide.

> Perhaps something to put as a TODO for whoever is in charge of marketing?

Sadly we haven't been able to fill that role for the past 2-3 years at least by this point.

> I made two new files on this repository. I just noticed that the logo is not the same in the PNGs and the SVG. The SVG logo is more of a strict "T" while the PNGs have a stylized "T". Did someone forget to commit work?

Unless I'm missing something the finished variant of the icon in the svg file is exactly the same as the PNG version:
![kuva](https://github.com/flathub/com.revolutionarygamesstudio.ThriveLauncher/assets/3796411/4954266f-3283-4881-8cab-a19df6ff3345)

Commit:
misc: Add branding colors

From button hover / idle.

I felt they were the best Thrive colors.

https://wiki.revolutionarygamesstudio.com/wiki/Visual_Style_Guide
